api/src/main/java/uk/gov/dhsc/htbhf/claimant/eligibility/EligibilityDecisionHandler.java

```diff
@@ -79,7 +82,7 @@ public void handleIneligibleDecisionForActiveClaim(Claim claim,
         if (shouldExpireActiveClaim(decision, previousPaymentCycle, currentPaymentCycle)) {
             expireClaim(claim, decision.getDateOfBirthOfChildren(), UPDATED_FROM_ACTIVE_TO_EXPIRED);
         } else if (decision.getIdentityAndEligibilityResponse().isNotEligible()) {
-            handleLossOfQualifyingBenefitStatus(claim, decision.getDateOfBirthOfChildren());
+            handleLossOfQualifyingBenefitStatus(claim, decision.getDateOfBirthOfChildren(), currentPaymentCycle);
```

Comment:
I suggest we call `paymentCycleService.updateEndDateForClaimBecomingActive(currentPaymentCycle);` directly instead of modifying `handleLossOfQualifyingBenefitStatus`

Reply:
Updated to call updateEndDateForClaimBecomingActive after handleLossOfQualifyingBenefitStatus